instrumentation/powerjob-4.0/javaagent/build.gradle.kts

```diff
@@ -0,0 +1,25 @@
+plugins {
+  id("otel.javaagent-instrumentation")
+}
+
+muzzle {
+  pass {
+    group.set("tech.powerjob")
+    module.set("powerjob-worker")
+    versions.set("[4.0.0,)")
+    assertInverse.set(true)
+    extraDependency("tech.powerjob:powerjob-official-processors:1.1.0")
+  }
+}
+
+dependencies {
+  library("tech.powerjob:powerjob-worker:4.0.0")
```

Comment:
![image](https://github.com/user-attachments/assets/de88cff9-4077-4486-a06e-ce6f6cb123d6)
![image](https://github.com/user-attachments/assets/107aeeb6-93f5-47fd-ab91-edeb46a24035)

i wonder why not create a new instrumentation for the latest version of powerjob?

Reply:
The current instrumentation is also compatible with the latest version of PowerJob. Using a lower version allows compatibility with more versions.